force-app/main/default/aura/autoEnrollmentMappingModalOpener/autoEnrollmentMappingModalOpenerHelper.js

```diff
@@ -139,30 +139,31 @@
         });
         modalSaveEvent.fire();
     },
-    /*
+
     handleModalDeleteConfirm: function (component) {
         let modalSaveEvent = component.getEvent("modalSaveEvent");
 
+        const actionName = component.get("v.actionName");
         const mappingName = component.get("v.mappingName");
-        const affiliationsAction = component.get("v.affiliationsAction");
-        const accountRecordType = component.get("v.accountRecordType");
-        const contactField = component.get("v.contactField");
-        const autoProgramEnrollment = component.get("v.autoProgramEnrollment");
+        const accountRecordType = component.get("v.oldAccountRecordType");
+        const autoProgramEnrollmentStatus = component.get("v.autoProgramEnrollmentStatus");
+        const autoProgramEnrollmentRole = component.get("v.autoProgramEnrollmentRole");
 
-        const saveModel = {
-            modalType: "affiliations",
-            action: affiliationsAction,
+        const deleteModel = {
+            modalType: "autoenrollmentmapping",
+            action: actionName,
             mappingName: mappingName,
-            accountRecordType: accountRecordType,
-            contactField: contactField,
-            autoEnrollmentEnabled: autoProgramEnrollment
+            oldAccountRecordType: accountRecordType,
+            autoProgramEnrollmentStatus: autoProgramEnrollmentStatus,
```

Comment:
We dont really need these parameters for deletion?

Reply:
Right, these are not needed. I will remove them

Reply:
I am still seeing these variables in the code.

Reply:
Strange, I removed them, here is the commit: https://github.com/SalesforceFoundation/EDA/pull/1759/commits/3f85ebd57cc3c6a942c58ccec2dfcb19ecc9dcfc